src/test/java/org/folio/entitlement/integration/keycloak/KeycloakServiceTest.java

```diff
@@ -73,11 +78,23 @@ class KeycloakServiceTest {
   @Mock private KeycloakModuleDescriptorMapper keycloakModuleDescriptorMapper;
   @Spy private final KeycloakConfigurationProperties kcConfiguration = keycloakConfigurationProperties(CLIENT_SUFFIX);
 
+  @Mock private KeycloakRetrySupportService retrySupportService;
   @Mock(answer = RETURNS_DEEP_STUBS) private Keycloak keycloak;
   @Mock(answer = RETURNS_DEEP_STUBS) private AuthorizationResource authorizationResource;
 
+  @BeforeEach
+  void setUp() {
+    lenient().when(retrySupportService.callWithRetry(any()))
```

Comment:
lenient usage usually not a good practice, use this stub directly in the test where multiple calls occurred

Reply:
It's not a question of multiple calls - the mock is used where there are certain kinds of calls.

I've replaced lenient with explicit calls as requested, but it looks ugly now TBH.